### ISSUE TITLE
Feature of providing response stub

### ISSUE THREAD

**Issue by szczja** [bacardi55/gtl#28]
I think that is less important to start editor from gtl, but gtl could provide some easy way for preparing some template for copy&pasting to editor running, for example, in second window. I thinking about something like browsing through tinylog, and get response stub (date, reference link, nickname, and place for our response). 

```
## <Current date in supported format>
=> <Base tinylog> Re: <Base date from selected post>
<Base nickname>: <Cursor position> or <Quoted base post>
```
For copy&pasting I suggest consider a new dialog with textfield which will be easy to copy in tmux. Rather than doing some advance solutions with xclip etc.

**Comment by bacardi55** [comment]
The next version of the TUI timeline should help implementing this, but it is not possible in the current state.

**Comment by bacardi55** [comment]
Available in branch timelinev2
Will be released with 0.6.0
